projects/storefrontlib/src/feature-toggles.ts

```diff
@@ -7,5 +7,6 @@ export interface FeatureToggles {
      */
     level?: string;
     consignmentTracking?: boolean;
+    qualtrics?: true;
```

Comment:
Let's remove this feature toggle. As long as we add features in a data driven way, there's no need to add  a feature flags. Feature flags are there to solve a problem – breaking changes; if we don't break something, we don't need it. Since it's a data driven solution, there's nothing we break.